ansible/ansible-tools/roles/knative_eventing/defaults/main.yaml

```diff
@@ -0,0 +1,5 @@
+---
+knative_eventing_crds: "https://github.com/knative/eventing/releases/download/v0.22.0/eventing-crds.yaml"
```

Comment:
It would make sense to extract version as separate variable but we can do it later